.github/workflows/continuous-integration.yml

```diff
@@ -61,11 +61,10 @@ jobs:
             fpmodel: SP
     env:
       FC: gfortran-${{ matrix.gfortran-version }}
-      FFLAGS: "-ffree-line-length-none -m64 -std=f2008 -march=native -fbounds-check -fmodule-private -fimplicit-none -finit-real=nan"
+      FFLAGS: "-m64 -std=f2008 -march=native -fbounds-check -fmodule-private -fimplicit-none -finit-real=nan"
```

Comment:
Noting here an exchange I had with @skosukhin, noting that only `gfortran` requires this flag.